managed-html5-runtime-basic-mta/mta.yaml

```diff
@@ -21,21 +21,6 @@ modules:
           service-key:
             name: hello-world_html_repo_host-key
       - name: hello-world-destination-service
-        parameters:
-          content:
```

Comment:
this should be moved to a new parameters element in the same module. e.g. after line 25.